src/app/store/technical-records/reducers/batch-create.reducer.ts

```diff
@@ -38,35 +38,35 @@ export const batchAdapter: EntityAdapter<BatchRecord> = createEntityAdapter<Batc
 export const initialBatchState: BatchRecords = batchAdapter.getInitialState({
   generateNumber: false
 });
+//TODO: V3 Batch to use V3 records
+// export const vehicleBatchCreateReducer = createReducer(
+//   initialBatchState,
+//   on(upsertVehicleBatch, (state, action) => batchAdapter.setAll(action.vehicles, state)),
+//   on(setGenerateNumberFlag, (state, { generateNumber }) => ({ ...state, generateNumber })),
+//   on(setApplicationId, (state, { applicationId }) => ({ ...state, applicationId })),
+//   on(setVehicleStatus, (state, { vehicleStatus }) => ({ ...state, vehicleStatus })),
+//   on(setVehicleType, (state, { vehicleType }) => ({ ...state, vehicleType })),
+//   on(createVehicleRecordSuccess, (state, action) => batchAdapter.updateOne(vehicleRecordsToBatchRecordMapper(action.vehicleTechRecords)[0], state)),
+//   on(updateTechRecordsSuccess, (state, action) =>
+//     batchAdapter.updateOne(vehicleRecordsToBatchRecordMapper(action.vehicleTechRecords, true, true)[0], state)
+//   ),
+//   on(clearBatch, state => batchAdapter.removeAll({ ...state, vehicleStatus: '', applicationId: '', vehicleType: undefined }))
+// );
 
-export const vehicleBatchCreateReducer = createReducer(
```

Comment:
are we not doing batch for LGV V3?

Reply:
No currently on develop batch is only for HGV, PSV and TRL